libvast/src/segment_store.cpp

```diff
@@ -482,7 +482,7 @@ uint64_t segment_store::drop(segment_builder& x) {
 }
 
 std::filesystem::path store_path_for_partition(const uuid& id) {
-  auto store_filename = fmt::format("{}.store", id);
+  auto store_filename = fmt::format("{:u}.store", id);
```

Comment:
It seems very risky to me to make this formatter change in a PR that I assume you intended to merge during the feature freeze. We need to carefully test this with older databases, or merge this only after the release.

Reply:
Let's wait until after the release then.

Reply:
I think we can rebase and merge this.